botocore/signers.py

```diff
@@ -196,6 +208,65 @@ def generate_presigned_url(self, request_dict, expires_in=3600,
         return request.url
 
 
+class CloudFrontSigner(object):
```

Comment:
But please don't forget that CloudFront not only allows signing URLs, they also supports [signing cookies](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-signed-cookies.html). I envision we may have a generate_signed_cookie() into this CloudFrontSigner class in (far?) future.